src/Microsoft.FeatureManagement/FeatureFilterConfiguration.cs

```diff
@@ -7,12 +7,12 @@
 namespace Microsoft.FeatureManagement
 {
     /// <summary>
-    /// The settings that define a feature filter.
+    /// The configuration of a feature filter.
     /// </summary>
-    class FeatureFilterSettings
+    public class FeatureFilterConfiguration
```

Comment:
I am wondering if it will be better if we call this `FeatureFilterDefinition`.

Reply:
The reason I left it as Configuration rather than definition is that there is only one definition for a feature. However the same feature filter might appear in multiple features. They would just be configured differently. So to me, this was the configuration of a feature filter rather than a definition of one.